.github/workflows/contributors-record.yml

```diff
@@ -0,0 +1,42 @@
+# Give your test a name!
+name: ContributorsRecord
+
+# runs daily
+on:
+  schedule:
+    - cron:  '*/1 * * * *'
```

Comment:
```suggestion
    - cron:  '* */23 * * *'
```

Reply:
OMG - you know me - I don't really read most of my GitHub notifications but this one made me laugh!! Good catch! When do you think we'll hit the event horizon and need to update the contributors page every minute 😉

Reply:
I can see in my notification how 1 min update can really bother people who have all notification on 😅